docs/extensions/guides/protocol-handlers.md

```diff
@@ -15,6 +15,23 @@ Handlers will be run in both `main` and `renderer` in parallel with no synchroni
 Furthermore, both `main` and `renderer` are routed separately.
 In other words, which handler is selected in either process is independent from the list of possible handlers in the other.
 
+Example of registering a handler:
+
+```typescript
+import { LensMainExtension, Interface } from "@k8slens/extensions";
+
+function rootHandler(params: Iterface.ProtocolRouteParams) {
+  console.log("routed to ExampleExtension", params);
+}
+
+export default class ExampleExtensionMain extends LensMainExtension {
+  protocolHandlers = [
+    pathSchema: "/",
+    handler: rootHandler,
+  ]
+}
+```
+
```

Comment:
You could also mention the specific URI that would result in calling rootHandler, like using
```
open "lens://extension/example_extension/
```
giving an end-to-end example